about-me.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="about-me.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <title>My Biography</title>
+
+</head>
+
+
+<body>
+
+    <header>
+        <h1>My Biography</h1>
+        
+        
+    </header> 
+
+    <nav class="navbar navbar-expand-lg navbar" style="background-color: lightgray;">
+        <div class="collapse navbar-collapse" id="navbarNav">
+
+            <ul class="navbar-nav">
+            <li class="nav-item active">
+            <a class="nav-link" href="file:///C:/Users/kty35/Desktop/Bootstrap-Portfolio/Bootstrap-Portfolio/about-me.html#">About Me <span class="sr-only">(current)</span></a>
+            </li>
```

Comment:
You linked your other pages wrong, it would be something like

```html
<a class="nav-link" href="/about-me.html">About Me <span class="sr-only">(current)</span></a>
```

Remember relative paths!